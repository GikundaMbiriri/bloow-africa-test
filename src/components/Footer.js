import React from 'react'

function Footer() {
  return (
    <div className='object-bottom'>

<footer className="bg-gray-200 text-center lg:text-left h-14 ">
  <div className="text-gray-700 text-center p-4">
    Â© 2021 Copyright:
    <a className="text-gray-800" href="https://tailwind-elements.com/">Bloow Africa</a>
  </div>
</footer>
    </div>
  )
}

export default Footer