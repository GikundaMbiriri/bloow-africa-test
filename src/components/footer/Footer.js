import React from 'react'
import {Link } from 'react-router-dom'
function Footer() {
  return (
    <div className='object-bottom'>

<footer className="bg-gray-200 text-center lg:text-left h-14 ">
  <div className="text-gray-700 text-center p-4">
    Â© 2022 Copyright:
    <a className="text-gray-800" href="/">Bloow Africa</a>
  </div>
</footer>
    </div>
  )
}

export default Footer