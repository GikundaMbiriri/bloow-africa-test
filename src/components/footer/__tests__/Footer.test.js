import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer';
import Footer from '../Footer'
it("it renders without crashing",()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Footer></Footer>,div)
})

it('renders correctly', () => {
    const tree = renderer
      .create(<Footer></Footer>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });