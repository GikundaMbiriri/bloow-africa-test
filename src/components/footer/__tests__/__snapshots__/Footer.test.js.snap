// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="object-bottom"
>
  <footer
    className="bg-gray-200 text-center lg:text-left h-14 "
  >
    <div
      className="text-gray-700 text-center p-4"
    >
      Â© 2022 Copyright:
      <a
        className="text-gray-800"
        href="https://tailwind-elements.com/"
      >
        Bloow Africa
      </a>
    </div>
  </footer>
</div>
`;
